returnn/tf/layers/basic.py

```diff
@@ -551,19 +551,18 @@ def __init__(self, activation, **kwargs):
       self.output.placeholder = self.output_before_activation.y
 
   @classmethod
-  def get_out_data_from_opts(cls, activation, **kwargs):
+  def get_out_data_from_opts(cls, sources, name, activation, **kwargs):
```

Comment:
The new arguments should also be documented.

Reply:
You are right, I added the documentations.